Dockerfile

```diff
@@ -13,16 +16,34 @@ RUN apt-get update && \
     rm -rf /var/lib/apt/lists/*
 
 # Copy the current directory contents into the container at /app
-COPY . /app
+COPY requirements.txt .
+
+# Make a virtualenv that we can copy across multistage builds
+ENV VIRTUAL_ENV=/venv
+RUN python -m venv $VIRTUAL_ENV
+# "Activate" the virtualenv
+ENV PATH="$VIRTUAL_ENV/bin:$PATH"
 
 # Install any needed packages specified in requirements.txt
-RUN pip wheel --no-cache-dir --wheel-dir=wheels -r requirements.txt
-RUN pip install --no-cache-dir --find-links=wheels -r requirements.txt
+RUN pip install --no-cache-dir -r /app/requirements.txt
```

Comment:
can you not remove `pip wheel` ? It speeds up installing requirements.txt 

cc @PSU3D0

Reply:
yes it speeds it up, but its just double handling, pip wheel is pulling the packages down then installing them the pip install is doing that in one step.